### DISCUSSION TITLE
How to Add System Variables / Update `PATH` in WinPython Environment

### DISCUSSION THREAD

**Discussion by RoyiAvital** [winpython/winpython#1009] [Q&A]
I want to manually edit / add some System / Environment Variables for WinPython.
Namely, I want to effect the process of Python so those variables are available to it.

Where is the correct place to set those?

**Comment by RoyiAvital** [comment]
OK, I found at the Wiki - [Environment](https://github.com/winpython/winpython/wiki/Environment).
But what if I want just to add a folder path to `PATH`? It seems it only allows overriding it.

**Reply by RoyiAvital** [reply]
I tried something like `PATH=%CUDA_PATH%\bin;%CUDA_PATH%\libnvvp;%PATH%` which seems to be resolved correctly.
Is that the official way to do so?

**Reply by stonebig** [reply]
all the WinPython environnement relies on setting the PATH (in `\scripts\env.bat`), so that looks pertinent.

**Reply by RoyiAvital** [reply]
The issue is, documentation doesn't say if one can use other / existing variables in `winpython.ini`.
If it was written, then I'd know I could append to the current path by: 

```
PATH = %PATH%; <MyFolderToAddToPath>;
```

I will add it to the Wiki page.
